### PR TITLE
Readme: update Windows building instructions

### PR DESCRIPTION
**Pull request by gkoz**
This better reflects the findings from #44

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ Make sure both mingw's and the sdk's `bin` directories are in your `PATH` e.g. (
 ```
 C:\> set PATH="C:\mingw-w64\bin;C:\gtk\bin;%PATH%"
 ```
-It's crucial that GCC from mingw is used by Rust so either make sure that mingw is earlier in the `PATH` or delete `gcc.exe` and `ld.exe` from the Rust installation.
+You will get a linking error `ld: cannot find -limm32` if `gcc.exe` and `ld.exe` that come with Rust are present in its `bin` directory so make sure they're not there.
```
Comment (GuillaumeGomez) [inline README.md]: "that come__s__", no ?

Comment (gkoz) [inline README.md]: There're two files so plural. I feel though that the sentence may not be clear enough so if you have better ideas, go for it =)

Comment (GuillaumeGomez) [inline README.md]: Hum... I didn't read it like that. I'd replace "that" by "which" in this case. But I'm not an english native (and maybe you are !), so ?

Comment (gkoz) [inline README.md]: I'm not, but I've looked into "that"/"which" question recently. They're completely equivalent, what matters is the presence or absence of a comma before "that".

**Comment by gkoz** [comment]
I'll try to improve the wording later today

**Comment by gkoz** [comment]
There.

**Comment by GuillaumeGomez** [comment]
Yes sir !

**Comment by GuillaumeGomez** [comment]
Way better like this, thanks !

**Pull request merged**